torch_np/tests/numpy_tests/lib/test_function_base.py

```diff
@@ -23,12 +22,12 @@
 IS_PYPY = False
 
 import numpy.lib.function_base as nfb   # FIXME: remove
-from numpy.random import rand    # FIXME: random.rand
+from torch_np.random import rand
 
 
 # FIXME: make from torch_np
```

Comment:
We could potentially do a `from torch_np import *` for now, to avoid having to change all those `average` into `np.average`.

Reply:
I'm avoiding star imports like a plague (the famous `from numpy import *; sum(a, 1)` thing), but yes, will mimic numpy old-fasioned  explicit imports to minimize the diff.